CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ This file documents all notable changes to the GEOS-Chem repository starting in
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] - TBD
+### Added
+- Added `#InvCEDSshipALK6` (commented out by default) to GCClassic `HEMCO_Diagn.rc.fullchem` and WRF `HEMCO_Diagn.rc` files
+- Added `EmisOCs*` diagnostics to `run/GCHP/HEMCO_Diagn.rc.templates/HEMCO_Diagn.rc.carbon` (these were missing)
+
+### Changed
+- Updated GCHP template files `HEMCO_Diagn.rc.fullchem` and `HISTORY.rc.fullchem` so that the same emission diagnostics are requested in both
+- Changed `ALD2_PLANTDECAY` emissions category from 3 to 99 so as not to conflict with the anthropogenic transport sector.
```

Comment:
Add that this is for GEOS-only

Reply:
Now done in commit 7e0bcc1297a5627b0c629437415fd5611369e51d